packages/arb-validator/challenges/messagesChallenge.go

```diff
@@ -179,19 +182,19 @@ func defendMessages(
 			return ChallengeAsserterWon, nil
 		}
 
+		inboxHashes := make([]value.HashOnlyValue, bisectionCount)
+		preImages := make([]common.Hash, bisectionCount)
```

Comment:
If `timedOut` is false, these will both be empty lists. In the case where a timeout has not occurred, that means that we are replaying this challenge and so we need to continue from the data in the `arbbridge.MessagesBisectionEvent`. Happy clarify/help figure out the solution if that's useful.